packages/kotti-ui/source/kotti-i18n/types.ts

```diff
@@ -31,6 +37,7 @@ export namespace KottiI18n {
 	export type Props = {
 		locale: SupportedLanguages
 		messages: DeepPartial<Messages>
+		numberFormat: NumberFormat
```

Comment:
Actually, we should probably allow this to be a DeepPartial for future extensibility